app/services/policies/policies_list_service.rb

```diff
@@ -61,4 +66,20 @@ def to_h
       @sets.transform_values(&:to_a).as_json
     end
   end
+
+  HTTP_ERRORS = [HTTPClient::BadResponseError, HTTPClient::TimeoutError, HTTPClient::ConnectTimeoutError, HTTPClient::SendTimeoutError, HTTPClient::ReceiveTimeoutError, SocketError].freeze
+  private_constant :HTTP_ERRORS
+
+  def self.fetch_policies_from_apicast
+    apicast_registry_url = ThreeScale.config.sandbox_proxy.apicast_registry_url
+    begin
+      response = ::JSONClient.get(apicast_registry_url)
```

Comment:
`ThreeScale.config.sandbox_proxy.apicast_registry_url` could be `nil` and I don't know if the error that it would raise in that case is being rescued here or not 🤔

Reply:
For that particular case it throws `SocketError`, which is rescued.